addons/stock/report/report_stock_quantity.py

```diff
@@ -73,13 +83,16 @@ def init(self):
         stock_quant q
     LEFT JOIN stock_location l on (l.id=q.location_id)
     LEFT JOIN stock_warehouse wh ON l.parent_path like concat('%/', wh.view_location_id, '/%')
+    LEFT JOIN product_product pp on pp.id=q.product_id
+    LEFT JOIN product_template pt on pt.id=pp.product_tmpl_id
```

Comment:
This left join seems not useful, you can already select on pp.product_tmpl_id. Right ?